common.tf

```diff
@@ -33,6 +33,8 @@ locals {
   enable_apiary_s3_log_management = var.apiary_log_bucket == "" ? true : false
   apiary_s3_logs_bucket           = local.enable_apiary_s3_log_management ? "${local.apiary_bucket_prefix}-s3-logs" : ""
   apiary_s3_hive_logs_bucket      = local.enable_apiary_s3_log_management ? "${local.apiary_s3_logs_bucket}-hive" : ""
+  apiary_system_bucket            = "${local.apiary_bucket_prefix}-${var.system_database_name}"
```

Comment:
can you replace `_` with `-` for bucket name, s3 doesn't support `_` in bucket name